source/_integrations/sms.markdown

```diff
@@ -61,29 +61,34 @@ Every time there is a message received, `event: sms.incoming_sms` is fired with
 Sample automation that forward all SMS to `user1`:
 
 #### Define a sensor in `configuration.yaml` to protect user phone number
+
+{% raw %}
```

Comment:
There are no jinja syntax templates in the example below, so it won't be needed to escape this (no liquid collision).

```suggestion
```